app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/state/EnabledMessageComposerStateHolder.kt

```diff
@@ -47,21 +46,15 @@ class EnabledMessageComposerStateHolder {
         }
     }
 
-    fun handleOffsetChange(offset: Dp, navBarHeight: Dp, focusManager: FocusManager) {
+    fun handleOffsetChange(offset: Dp, navBarHeight: Dp) {
```

Comment:
question: do we need a `navBarHeight`? Can't we just apply `navigationBarsPadding` to the `AdditionalOptionSubMenu`?

Reply:
It doesn't help because I was trying to solve it from compose side. I would leave entire logic for height here